lms/templates/credit_notifications/credit_eligibility_email.html

```diff
@@ -30,10 +30,12 @@
             </p>
 
             <p>
-              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your edX course! Click {link} to get your credit now.").format(
-                link=u'<a href="{dashboard_url}">here</a>'.format(
+              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your {platform_name} course! Click {link_start}here{link_end} to get your credit now.").format(
```

Comment:
Can we change this from `Click {link_start}here{link_end} to get your credit now` to `{link_start}Click here to get your credit now{link_end}`? It's more accessible when the whole phrase is linked.